src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
It's better to use more meaningful variable names

```suggestion
  const men = century
    ? people.filter((person) => person.sex === 'm' && Math.ceil(person.died / 100) === century)
    : people.filter((person) => person.sex === 'm');
```